dev-qt/qtwebengine/qtwebengine-5.11.9999.ebuild

```diff
@@ -15,17 +16,18 @@ IUSE="alsa bindist designer geolocation +jumbo-build pax_kernel pulseaudio
 	+system-ffmpeg +system-icu widgets"
 REQUIRED_USE="designer? ( widgets )"
 
+QT_PV="5.11.1"
```

Comment:
Maybe `QT_PV="5.11.1:="` would be better, since in QtWebEngine code there are possibly some `#if` conditionals with versions of other Qt packages...

Reply:
probably yes, however it is not yet clear if we can make it work with revdeps anyway. it seems that qtwebengine-installed cmake config insists on same-version qt dependencies still, don't know yet how Fedora and Debian do it.